package.json

```diff
@@ -5,19 +5,19 @@
     "packages/*"
   ],
   "dependencies": {
-    "react": "^16.8.5",
-    "react-dom": "^16.8.5"
+    "react": "^16.13.1",
```

Comment:
@jennyscript could we keep the other `package.json` files also in sync with these updates?

![image](https://user-images.githubusercontent.com/127199/79520614-1c3dda80-800c-11ea-9b0f-bdc7acea8677.png)